tutorials/02-functions/02-what-is-this.md

```diff
@@ -355,7 +366,7 @@ var person = {
   }
 };
 
-person.updateName();
+person.updateName(); // Maybe we're fetching the new name from the server.
```

Comment:
Given the fact that we're using comments to describe the code, I don't like that we're also using them to show a side-effect (the `console.log` in this case).